packages/auth/lib/index.d.ts

```diff
@@ -1722,6 +1773,9 @@ export namespace FirebaseAuthTypes {
      */
     signInWithCredential(credential: AuthCredential): Promise<UserCredential>;
 
+    signInWithPopup(provider: AuthProvider): Promise<UserCredential>;
+    signInWithRedirect(provider: AuthProvider): Promise<UserCredential>;
```

Comment:
Can you document this here, even if briefly? The rnfirebase.io site is built from the typedoc so anything you put here will be seen + used